packages/app-studio-toolkit/src/basctlServer/basctlServer.ts

```diff
@@ -64,7 +66,9 @@ function createBasctlServer() {
   }
 }
 
-export function startBasctlServer() {
+export function startBasctlServer(context: vscode.ExtensionContext) {
+  // add the socket file name to the environment variables so that the basctl client can use it
```

Comment:
Maybe add a link to the relevant place where it is used? e.g:
https://github.wdf.sap.corp/devx-wing/bas-cli/blob/master/src/socketClient.js#L4

Reply:
it is an OSS and the link you mentioned will be unreachable externally